### ISSUE TITLE
Dashboard is broken

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#690]
Probably related to the update of django for the PKI ?

Tried restarting graphite and it didn't do anything

Steps made : 
- Install 5.1
- Upgrade to 5.2
- Configure PF and start it

Stacktrace : 

```
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99] mod_wsgi (pid=3260): Exception occurred processing WSGI script '/usr/local/pf/conf/httpd.conf.d/graphite-web.wsgi'.
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99] Traceback (most recent call last):
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]   File "/usr/lib/python2.6/site-packages/django/core/handlers/wsgi.py", line 206, in __call__
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]     response = self.get_response(request)
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]   File "/usr/lib/python2.6/site-packages/django/core/handlers/base.py", line 194, in get_response
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]     response = self.handle_uncaught_exception(request, resolver, sys.exc_info())
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]   File "/usr/lib/python2.6/site-packages/django/core/handlers/base.py", line 232, in handle_uncaught_exception
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]     if resolver.urlconf_module is None:
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]   File "/usr/lib/python2.6/site-packages/django/core/urlresolvers.py", line 360, in urlconf_module
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]     self._urlconf_module = import_module(self.urlconf_name)
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]   File "/usr/lib/python2.6/site-packages/django/utils/importlib.py", line 40, in import_module
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]     __import__(name)
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]   File "/usr/lib/python2.6/site-packages/graphite/urls.py", line 15, in <module>
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99]     from django.conf.urls.defaults import *
[Fri Jun 12 17:50:04 2015] [error] [client 172.20.20.99] ImportError: No module named defaults
```

**Comment by fdurand** [comment]
yum update graphite-web --enablerepo=packetfence-devel

**Comment by julsemaan** [comment]
worked. Should be added to dependencies then ?

**Comment by fdurand** [comment]
We have to specify a version

**Comment by fdurand** [comment]
Fixed in 8e18f2558a5400831c654788648599a62baa8b11
